src/content/feed.njk

```diff
@@ -10,15 +10,13 @@ permalink: feed.xml
   <id>{{ markup.baseURL }}/</id>
   {%- for post in posts %}
   {% set absolutePostURL %}{{ post.permalink | url | absoluteUrl(markup.baseURL) }}{% endset %}
-  {% set authorList = authors | postAuthors(post.author) %}
+  {% set author = authors | getAuthor(post.author) %}
   <entry>
     <title>{{ post.title }}</title>
     <link href="{{ absolutePostURL }}" />
     <updated>{{ post.modified | convertToJsDate | dateToRfc3339 }}</updated>
     <id>{{ absolutePostURL }}</id>
-    {%- for author in authorList %}
```

Comment:
Is it not possible to have more than one author? In the past on mozilla hacks there's been co-authored posts.

Reply:
That's something I should have mentioned: the api response looks like it isn't possible so I don't know.

Reply:
The editor seems to enforce a single author.